README.md

```diff
@@ -52,10 +65,11 @@ The following options can be passed to the plugin via `docker plugin set` (all n
 - **`apikey`** (**required**): authentication token to use when accessing the Hetzner Cloud API
 - **`size`** (optional): size of the volume in GB (default: `10`)
 - **`fstype`** (optional): filesystem type to be created on new volumes. Currently supported values are `ext{2,3,4}` and `xfs` (default: `ext4`)
-- **`prefix`** (optional): prefix to use when naming created volumes (default: `docker`)
+- **`prefix`** (optional): prefix to use when naming created volumes; it will be separated from the created volume name with a hyphen (default: `docker`)
```

Comment:
maybe we could be even clearer with an example?
```suggestion
- **`prefix`** (optional): prefix to use when naming created volumes; the final name on the HC side will be of the form `prefix-name`, where `name` is the volume name assigned by `docker` (default: `docker`)
```

Reply:
Good suggestion! :+1: